README.rst

```diff
@@ -46,7 +46,21 @@ To build the PDF form of the documentation simply execute the following command:
 
   make latexpdf
 
-Note: Creating the PDF form of the documentation requires that latexmk is available, information on latexmk can be found at https://ctan.org/pkg/latexmk/.
+Note: Creating the PDF form of the documentation requires that some or all of a tool like Windows/MikTeX, macOS/MacTeX, Windows/TeXlive, Linux/TeXlive is available to the Sphinx build tool.
```

Comment:
I would put things in a slightly different order (and not use a capital letter after a column):

Note: to create the PDF form of the documentation requires that some or all of a tool like Windows/MikTeX, Windows/TeXlive, Linux/TeXlive and macOS/MacTeX is available to the Sphinx build tool.